src/ArtistList.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import ArtistDetail from './ArtistDetail';
+import Artist from './Artist';
+
+class ArtistList extends React.Component{
+    state ={
+        sortType: '',
+        artists: []
+    }
+
+    componentDidMount() {
+        this.setState({
+            artists: ArtistDetail
+        })
+    }
+
+
+    //Vote handler
+    voteHandler = (artistId) => {
+        const voteArtist = this.state.artists.map(artist => {
+            if(artistId === artist.id) {
+                return Object.assign({}, artist, {
+                    votes: artist.votes + 1
+                })
```

Comment:
You can use the spread operator instead of `Object.assign` as follows:
```suggestion
                return {
                    ...artist,
                    votes: artist.votes + 1
                }
```